### PR TITLE
V1.4.1 - Update Documentations

### PR DESCRIPTION
**Pull request by marcustw**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1250,7 +1250,7 @@ Use case ends.
 ### 10.5. Glossary
 
 * **Mainstream OS**: Windows, Linux, Unix, OS-X
```
Comment (nweiyue) [inline docs/DeveloperGuide.md]: Perhaps a full stop at the end here as well?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -157,13 +160,14 @@ For the good majority of visual learners, we figured to present information in a
...
    ![StartPage](images/StartPage.png)
+   <p align="center"> <sub> <b>Figure 3.1</b>: Example of start page </sub> </p>
 
 5. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
    Some example commands you can try:
 
    * **`liststu`** : Lists out all the students.
 
-   * **`addstu n/John Cena m/A0123456X e/thechamp@u.nus.edu`** : Adds a student named `John Cena` to the student list.
+   * **`addstu n/Jay Chou m/A0123456X e/jaychou@u.nus.edu`** : Adds a student named `Jay Chou` to the student list.
```
Comment (nweiyue) [inline docs/UserGuide.md:176]: Haha I thought we using Elvin Ng? But I also made changes to this. Will push later.

**Review by nweiyue — APPROVED** [review]
LGTM. Just a nit

File: docs/UserGuide.md (hunk 2 above)
Comment (marcustw) [reply docs/UserGuide.md:176]: Alright no problem!

**Pull request merged**